cluster-api/providers/aws/go.mod

```diff
@@ -0,0 +1,103 @@
+module openshift/installer/cluster-api/providers/aws
+
+go 1.20
+
+require sigs.k8s.io/cluster-api-provider-aws/v2 v2.2.4
+
+replace sigs.k8s.io/cluster-api-provider-aws/v2 => github.com/vincepri/cluster-api-provider-aws/v2 v2.0.0-20231013205918-8347c5355060
```

Comment:
I'm fine with merging these in the current state, but in the future, we would  use forks here, when appropriate, right?

Reply:
That's correct yes, we're have a couple of PRs left, this can be removed soon